travis/unit_tests/.gitignore

```diff
@@ -1,4 +1,3 @@
-Makefile
+# Ignore FRUIT files in this directory
 *_gen.f90
 fruit_driver.exe
-*.xml
```

Comment:
is this causing the error in codecov?

Reply:
No, I've changed a setting elsewhere and it's had un unexpected effect. I'm looking into it.